cinema/permissions.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
+
+
+class IsAdminOrIfAuthenticatedReadOnly(BasePermission):
```

Comment:
Don't you wanna use it in your views ?

Reply:
I use it in views, just do it through the definitions of the default permission.
![image](https://user-images.githubusercontent.com/46852739/232224118-ea216eee-5ce2-487f-9259-a690b089e740.png)